pyfixest/estimation/feols_.py

```diff
@@ -1384,8 +1384,8 @@ def fixef(self) -> dict[str, dict[str, float]]:
         D2 = Formula("-1+" + fixef_fml).get_model_matrix(_data, output="sparse")
         cols = D2.model_spec.column_names
```

Comment:
`lsqr` requires `uhat` to be a 1D array. You can solve many of the test error by changing 

```py
uhat = csr_matrix(Y - X @ self._beta_hat).transpose()
```
to 
```py
uhat = (Y - X @ self._beta_hat).flatten()
```